packages/components/CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 # @hashicorp/design-system-components
 
-## 3.5.1
+## 3.6.0
 
-### Patch Changes
+### Minor Changes
 
 **🔄 Updated dependencies:**
 
 - @hashicorp/design-system-tokens@1.11.0
-- @hashicorp/ember-flight-icons@4.0.7
+- @hashicorp/ember-flight-icons@4.1.0
+
+_Since this is an update brand colors and product icons, we consider this a `major` version release_
```

Comment:
I don't think it's clear what this note refers to – both `@hashicorp/design-system-components` and `@hashicorp/ember-flight-icons` are `minor`